src/main/webapp/app/pages/application/application-creation/application-creation-page1/application-creation-page1.component.ts

```diff
@@ -0,0 +1,156 @@
+import { CommonModule } from '@angular/common';
+import { Component, OnInit, model, output } from '@angular/core';
+import { StringInputTemporaryComponent } from 'app/shared/components/atoms/string-input-temporary/string-input-temporary.component';
+import { ApplicationForApplicantDTO } from 'app/generated';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { DividerModule } from 'primeng/divider';
+
+import { DropdownComponent, DropdownOption } from '../../../../shared/components/atoms/dropdown/dropdown.component';
+import { DatePickerComponent } from '../../../../shared/components/atoms/datepicker/datepicker.component';
+
+export type ApplicationCreationPage1Data = {
+  firstName: string;
+  lastName: string;
+  email: string;
+  phoneNumber: string;
+  gender?: DropdownOption;
+  nationality?: DropdownOption;
+  language?: DropdownOption;
+  dateOfBirth: string;
+  website: string;
+  linkedIn: string;
+  street: string;
+  streetnumber: string;
+  city: string;
+  country: string;
+  postcode: string;
+};
+
+export const dropdownGender: DropdownOption[] = [
+  { value: 'female', name: 'female' },
```

Comment:
I would suggest making the values capitalised, to maintain consistency with the rest of the dropdown values.
```suggestion
  { value: 'Female', name: 'female' },
```